src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({ sender, body, timeStamp, isLiked, id, updateEntry }) => {
+  const updateLike = () => {
+    updateEntry({
+      sender: sender,
+      body: body,
+      timeStamp: timeStamp,
+      liked: !isLiked,
+      id: id,
+    });
+    // console.log('running updateLike');
+  };
+
+  const heart = isLiked ? '❤️' : '🤍';
+  const locationClass = id % 2 === 0 ? 'remote' : 'local';
```

Comment:
Nice usage of the remainder operator here! This approach totally works, but it does assume that users alternate sending messages. Consider scenarios where users send multiple messages at a time. How could you refactor your code to handle this situation?